contracts/contracts/RequestManager.sol

```diff
@@ -138,6 +208,16 @@ contract RequestManager is Ownable {
         resolutionRegistry = ResolutionRegistry(_resolutionRegistry);
     }
 
+    /// Create a new transfer request.
+    ///
+    /// @param targetChainId ID of the target chain.
+    /// @param sourceTokenAddress Address of the token contract on the source chain.
+    /// @param targetTokenAddress Address of the token contract on the target chain.
+    /// @param targetAddress Recipient address on the target chain.
+    /// @param amount Amount of tokens to transfer. Does not include fees.
+    /// @param validityPeriod The number of seconds the request is to be considered valid.
+    ///                       A request cannot be fulfilled once its validity period has elapsed.
```

Comment:
Maybe just change it to 

`A request cannot be claimed once its validity period has elapsed`